pkgs/games/badlion-client/default.nix

```
@@ -25,7 +27,7 @@ in
       description = "The Most Complete All-In-One Mod Library for Minecraft with 100+ Mods, FPS Improvements, and more";
       homepage = "https://client.badlion.net";
       license = with licenses; [ unfree ];
-      maintainers = with maintainers; [];
-      platforms = platforms.linux;
+      maintainers = with maintainers; [ ];
+      platforms = [ "x86_64-linux" ];
```

```suggestion
      maintainers = with maintainers; [ ];
      platforms = platforms.linux;
```

@tie apparently tested this on non-x86_64-linux

---

How is that supposed to work when `src` is an ELF x86-64 executable file?
Sure, appimageTools doesn’t really care about that, but the resulting wrapper does not work for me.
```console
$ ./result/bin/badlion-client 
/nix/store/aqw62r6cx34hs5js1dbj62qplb96cjmp-badlion-client-3.15.0-extracted/AppRun: line 43: /nix/store/aqw62r6cx34hs5js1dbj62qplb96cjmp-badlion-client-3.15.0-extracted/BadlionClient: cannot execute binary file: Exec format error
/nix/store/aqw62r6cx34hs5js1dbj62qplb96cjmp-badlion-client-3.15.0-extracted/AppRun: line 43: /nix/store/aqw62r6cx34hs5js1dbj62qplb96cjmp-badlion-client-3.15.0-extracted/BadlionClient: Success
$ file /nix/store/aqw62r6cx34hs5js1dbj62qplb96cjmp-badlion-client-3.15.0-extracted/BadlionClient
/nix/store/aqw62r6cx34hs5js1dbj62qplb96cjmp-badlion-client-3.15.0-extracted/BadlionClient: ELF 64-bit LSB pie executable, x86-64, version 1 (SYSV), dynamically linked, interpreter /lib64/ld-linux-x86-64.so.2, for GNU/Linux 3.2.0, BuildID[sha1]=de95722a2dd82ddae1c67bb86600eab2733a2b6e, not stripped
```

---

Ah I'm sorry! I misinterpreted your earlier comments, thinking that "not tested on x86_64-linux desktop" means that you tested it on another platform, rather than just not on a desktop :smile: 

All good then!